hermes/probe/create/create.go

```diff
@@ -35,85 +38,84 @@ import (
 )
 
 const (
-	FileNamePrefixSize      = 9    // len("Hermes_ID")
-	begin                   = 1    // begin is the ID of the first HermesFile.
-	end                     = 50   // end is the ID of the last HermesFile.
-	maxFileSizeBytes        = 1000 // maxFileSizeBytes is the maximum allowed file size in bytes.
+	FileNameFormat          = "Hermes_%02d_%x"
+	minFileID               = 1
+	maxFileID               = 50
+	maxFileSizeBytes        = 1000
 	hermesAPILatencySeconds = "hermes_api_latency_s"
 )
 
 type randomFile struct {
-	id   int32 // id is a positive integer in the range [begin,end]
-	size int   // size is the file size in bytes.
+	id        int32
+	sizeBytes int
 }
 
 type randomFileReader struct {
-	size int // size in bytes
-	i    int // currently reading this byte
+	sizeBytes int
+	// currently reading this byte
+	i    int
 	rand *rand.Rand
 }
 
-// Read implements the standard Read interface, it reads data from a randomFileReader.
+// Read implements the standard Read interface, it returns a random stream of bytes generated by a pseudo-random generator.
 // Argument:
-//      buf: a byte slice serves as a buffer
+//      buf: a byte slice serves as an output buffer
 // Returns:
-//	n:  the size of the buffer buf following a Read operation
-//	err: error it should be nil when not done reading and io.EOF once the whole file contents have been read
+//		n:  the number of bytes read
+//		err: error it should be nil when not done reading and io.EOF once the whole file contents have been read
 func (r *randomFileReader) Read(buf []byte) (n int, err error) {
 	// check whether Read is done
-	if r.i >= r.size {
+	if r.i >= r.sizeBytes {
 		return 0, io.EOF
 	}
 	b := buf
-	if len(buf) > (r.size - r.i) {
-		// if the length of buffer is greater than the number of bytes left to read make the size of the buffer match the number of bytes left to read
-		b = buf[:(r.size - r.i)]
+	if len(buf) > (r.sizeBytes - r.i) {
+		// if the length of buffer is greater than the number of bytes left to read make the sizeBytes of the buffer match the number of bytes left to read
+		b = buf[:(r.sizeBytes - r.i)]
```

Comment:
Nit: there's no need for the parentheses.

Reply:
parantheses removed